README.md

```diff
@@ -1,4 +1,25 @@
 Le site de l'association Agile France
 =====================================
 
-Ce site est purement statique, et peut être édité directement en tant que HTML.
+## Développement
+
+Ce site est construit avec [Jekyll](https://jekyllrb.com/), un générateur de sites statiques. La version utilisée est celle [actuellement en production](https://pages.github.com/versions/) sur GitHub Pages.
+
+Pour l'améliorer, installer [Ruby](https://www.ruby-lang.org/fr/) et [Jekyll](https://jekyllrb.com) dans leurs [versions de production](https://pages.github.com/versions/) :
+
+```sh
+git clone https://github.com/betagouv/beta.gouv.fr.git
+cd beta.gouv.fr
+gem install bundler --no-ri --no-rdoc
```

Comment:
I had to do `sudo chown -R david /Library/Ruby/Gems/2.3.0/` not sure if it's because of my config, otherwise I had:

```
Fetching: bundler-1.16.1.gem (100%)
ERROR:  While executing gem ... (Errno::EACCES)
    Permission denied @ rb_sysopen - /Library/Ruby/Gems/2.3.0/cache/bundler-1.16.1.gem
```